apps/apps.humanatlas.io/src/app/pages/us1/cell-population-predictor/cell-population-predictor.component.html

```diff
@@ -0,0 +1,53 @@
+<div class="container">
+  <!-- HEADING -->
+  <div class="heading-container">
+    <div>
+      <img [src]="'images/cell-population-predictor-icon.svg' | assetUrl" alt="icon for cell population predictor" />
+    </div>
+    <div class="title-container">
+      <h1>Cell Population Predictor</h1>
+      <hra-software-status-indicator [status]="'Preview'" [size]="'medium'"></hra-software-status-indicator>
+    </div>
+  </div>
+
+  <!-- INFO -->
+  <div class="info-text">
+    <p>
+      Select a 3D tissue extraction site and use the Human Reference Atlas API to predict cell type populations for this
+      site.
+    </p>
+  </div>
+
+  <!-- CARDS -->
+  <div class="cards">
+    <hra-workflow-card title="Upload" class="card card-1">
+      <hra-workflow-card-actions>
+        <mat-icon aria-hidden="false" aria-label="info icon" class="info-icon" fontIcon="info"></mat-icon>
+      </hra-workflow-card-actions>
+      <div>
+        <p class="card-text">
+          Upload an existing extraction site made with the Registration User Interface (RUI) in the JSON format or use
+          the sample file.
+        </p>
+      </div>
+      <div class="buttons">
+        <button mat-button [disabled]="false" hraButtonSize="medium">Use Sample</button>
+        <button mat-button [disabled]="false" hraButtonSize="medium">Upload File</button>
+      </div>
+    </hra-workflow-card>
+    <hra-workflow-card title="Create" class="card card-2">
+      <hra-workflow-card-actions>
+        <mat-icon aria-hidden="false" aria-label="info icon" class="info-icon" fontIcon="info"></mat-icon>
```

Comment:
Icon needs to display a tooltip when hovered. Talk to @LibbyUX about what the tooltip content should be.